lib/jormungandr/src/Cardano/Wallet/Jormungandr.hs

```diff
@@ -116,27 +120,48 @@ serveWallet (cfg, sb, tr) databaseDir listen lj beforeMainLoop = do
             let nPort = Port $ baseUrlPort $ _restApi cp
             waitForService "Jörmungandr" (sb, tr) nPort $
                 waitForNetwork nl defaultRetryPolicy
-            apiLayer tr nl >>= startServer tr nPort nl
+            apiLayer tr (toWLBlock <$> nl) >>= startServer tr nPort nl
             pure ExitSuccess
         Left e -> handleNetworkStartupError e
   where
     startServer
         :: Trace IO Text
         -> Port "node"
-        -> NetworkLayer IO t (Block Tx)
+        -> NetworkLayer IO t Binary.Block
```

Comment:
We use to call `Binary.Block` -> `J.Block` everywhere else, so let's stick with that :+1: